tests/testthat/test_environment.R

```diff
@@ -11,7 +11,7 @@ test_that("create environment and check parameters", {
   expect_equal(env$name, env_name)
   expect_equal(env$version, "1")
   expect_equal(env$docker$enabled, TRUE)
-  expect_equal(env$docker$base_image, NULL)
+  expect_equal(env$docker$base_dockerfile, NULL)
```

Comment:
I believe this should be `env$docker$base_image <- custom_docker_image`. You no longer have `docker$base_dockerfile` as an attribute in your `r_environment` constructor.

Reply:
the base_image check is done in the test after this one. Since, in this test, the base image would be the default one, I didn't want to add the matching for a hardcoded string.